airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -105,7 +132,7 @@ class TIHeartbeatInfo(BaseModel):
 class TaskInstance(BaseModel):
     """Schema for TaskInstance model with minimal required fields needed for Runtime."""
 
-    id: uuid.UUID
+    id: uuid.UUID | int = 0
```

Comment:
This can't be 0

Reply:
Yeah, removed it. Had added for some initial changes, forgot to remove